packages/twenty-e2e-testing/tests/companies.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { test, expect } from '@playwright/test';
+import { config } from 'dotenv';
+import path = require('path');
+config({ path: path.resolve(__dirname, '..', '.env') });
+
+const date = new Date();
+
+test.beforeEach(async ({ page }) => {
```

Comment:
Should we extract this in a login module as you proposed or do you plan to do it in another PR ?

Reply:
I'll do it another PR